source/traefik_proxy.go

```diff
@@ -269,7 +269,6 @@ func (ts *traefikSource) ingressRouteEndpoints() ([]*endpoint.Endpoint, error) {
 		}
```

Comment:
Shell we have a test fir ipv4 and ipv6?

Reply:
This source does not seem to go into details about ipv4 and ipv6 in testing. The test I removed was purely about if or not the annotation was properly attached and producing a label or not.